packages/babel-jest/src/index.js

```diff
@@ -90,6 +90,8 @@ const createTransformer = (options: any) => {
         .update('\0', 'utf8')
         .update(fileData)
         .update('\0', 'utf8')
+        .update(filename)
```

Comment:
Can we make this relative to the rootDir (if it isn't already)?

Reply:
It's not, checking 😄

Reply:
In order to make it relative to rootDir we'll have to pass it here first:
https://github.com/facebook/jest/blob/0cb03f6474f52964ed74eadcde2ae3e219b4d04e/packages/jest-runtime/src/script_transformer.js#L77-L80

Reply:
Fine with me.

Reply:
Gonna make it relative in a minute.

Reply:
Pushed it, I also changed a flowtype hint as it appeared to be incorrect: https://github.com/facebook/jest/pull/4432/commits/69456cafda5056d72df88c68412d211abca05169#diff-066bbe3aae5baf14d3b62017f0dc86d4R85

According to the definition here: https://github.com/facebook/jest/blob/0cb03f6474f52964ed74eadcde2ae3e219b4d04e/types/Transform.js#L37-L41

getCacheKey gets `CacheKeyOptions` not `TransformOptions `.